.github/workflows/gh-ci-cron.yaml

```diff
@@ -196,3 +199,70 @@ jobs:
     - name: run_tests
       run: |
         pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+        
+  # Issue 4208
+  conda-latest-release:
+    # A set of runner to check that the latest conda release works as expected
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+        fail-fast: false
+        matrix:
+          os: [ubuntu-latest, macos-latest]
+          python-version: ["3.9", "3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: setup_micromamba
+      uses: mamba-org/setup-micromamba@v1
+      with:
+        environment-name: mda
+        create-args: >-
+          python=${{ matrix.python-version }}
+          pip
+        # using jaime's shim to avoid pulling down the cudatoolkit
+        condarc: |
+          channels:
+            - jaimergp/label/unsupported-cudatoolkit-shim
+            - conda-forge
+            - bioconda
+
+    - name: install_deps
+      uses: ./.github/actions/setup-deps
+      with:
+        micromamba: true
+        full-deps: true
```

Comment:
note to awake self - this doesn't work because this action already installs GSD which then clashes with the <3.0 requirement